ui-modules/utils/quick-launch/quick-launch.html

```diff
@@ -39,7 +39,8 @@ <h4 class="media-heading">{{app.name || app.symbolicName}}</h4>
             <h3 class="quick-launch-section-title">Name</h3>
             <div class="quick-launch-section-content">
                 <div class="form-group">
-                    <input class="form-control" type="text" ng-model="model.name" ng-disabled="deploying" name="name" placeholder="Choose a name for this application (Optional)" autofocus />
+                    <input class="form-control" type="text" ng-model="model.name" ng-disabled="deploying" name="name" 
+                        placeholder="{{ 'Choose a name for this application (Optional)' }}" />
```

Comment:
It's fine to remove the `autofocus` from this field but at least one in the form should have the `autofocus` directive assigned

Reply:
for me using it, it seems best not to have any with `autofocus` -- see below